firmware/application/apps/ui_settings.hpp

```diff
@@ -799,6 +799,41 @@ class SetMenuColorView : public View {
     };
 };
 
+class SetAutostartView : public View {
+   public:
+    SetAutostartView(NavigationView& nav);
+
+    void focus() override;
+
+    std::string title() const override { return "Autostart"; };
+
+   private:
+    int32_t i = 0;
```

Comment:
uint8_t should be enough no ?

Reply:
Should, but it is int32, because option_t uses the same, so don't have to cast:
using name_t = std::string;
using value_t = int32_t;
using option_t = std::pair<name_t, value_t>;